TRIAGING.md

```diff
@@ -0,0 +1,71 @@
+
+### What if I have follow-up questions on an issue?
+
+If you have an existing issue you would like to discuss, you can always comment on the issue itself. Alternatively, you are welcome to come to the triage meeting to discuss.
+
+### Is this meeting a good place to get help setting up features on my OpenSearch instance?
+
+While we are always happy to help the community, the best resource for implementation questions is [the OpenSearch forum](https://forum.opensearch.org/).
+
+There you can find answers to many common questions as well as speak with implementation experts.
+
+### What are the issue labels associated with triaging?
+
+Yes, there are several labels that are used to identify the 'state' of issues filed in OpenSearch .
+| Label | When applied | Meaning |
+| ----- | ------------ | ------- |
+| Untriaged | When issues are created or re-opened. | Issues labeled as 'Untriaged' require the attention of the repository maintainers and may need to be prioritized for quicker resolution. It's crucial to keep the count of 'Untriaged' labels low to ensure all potential security issues are addressed in a timely manner. See [SECURITY.md](https://github.com/opensearch-project/OpenSearch/blob/main/SECURITY.md) for more details on handling these issues. |
+| Help Wanted | Anytime. | Issues marked as 'Help Wanted' signal that they are actionable and not the current focus of the project maintainers. Community contributions are especially encouraged for these issues. |
+| Good First Issue | Anytime. | Issues labeled as 'Good First Issue' are small in scope and can be resolved with a single pull request. These are recommended starting points for newcomers looking to make their first contributions. |
+
+### What kind of outcomes will a triaged issue have?
+
+| Outcome | Label | Description | Canned Response | 
+| ------- | ----- | ----------- | --------------- |
+| Accepted | -`untriaged` | The issue has the details needed to be directed towards area owners. | "Thanks for filing this issue, please feel free to submit a pull request." |
+| Rejected | N/A | The issue will be closed with a reason for why it was rejected.  Some reasons will be due to lack of details, or being outside the scope of the project. | "Thanks for creating this issue; however, it isn't being accepted due to {REASON}.  Please feel free to re-open after addressing the reason." |
+| Area Triage | +`{AREA-LABEL}` | OpenSearch has many different areas and knowing if an issue should be accepted or not might not be possible with the representatives in the triage meeting, it will be labeled with the area and an owner will be @mentioned to follow up | "Thanks for creating this issue; the triage meeting was unsure if this issue should be accepted, @{PERSON} or someone from the area please review and then accept or reject this issue?" |
```

Comment:
Area owners - what do you think of this?  IMO this is a low friction way to delegate out deep dives or subject matter experts to consult on issues in areas they own.

Reply:
Maybe this is a good use case for teams? each area has a team and each team figures out how they're going to address the "Area Triage."

Reply:
opensearch-project's organization does not allow non-amazon employees, which means that none of contributors can be included in these org based groups.  :'(

I'd love to use teams if we could include everyone.